packages/flutter/lib/src/painting/fractional_offset.dart

```diff
@@ -8,10 +8,11 @@ import 'package:flutter/foundation.dart';
 
 import 'basic_types.dart';
 
-/// An offset that's expressed as a fraction of a Size.
+/// An offset that's expressed as a fraction of a [Size].
 ///
-/// FractionalOffset(1.0, 0.0) represents the top right of the Size,
-/// FractionalOffset(0.0, 1.0) represents the bottom left of the Size,
+/// `FractionalOffset(1.0, 0.0)` represents the top right of the [Size].
+///
+/// `FractionalOffset(0.0, 1.0)` represents the bottom left of the [Size].
```

Comment:
maybe also mention that the values can be <0, >1 to represent offscreen/outside [size] locations